src/LetsEncrypt/Internal/AcmeCertificateLoader.cs

```diff
@@ -117,11 +117,9 @@ private async Task LoadCerts(CancellationToken cancellationToken)
 
             var errors = new List<Exception>();
 
-            using var factory = new CertificateFactory(_options, _challengeStore, _logger, _hostEnvironment);
```

Comment:
`CertificateFactory` isn't thread safe or intended to have a long lifetime...part of the reason I didn't make it a singleton in the first place. I think there is a chance some of those assumptions are evident in the implementation of CertificateFactory and could be problematic due to this change....but I'm not 100% because I didn't write thorough tests (#6). Might be safer to avoid changing too much right now.